fdb-relational-core/src/test/java/com/apple/foundationdb/relational/SqlInsertTest.java

```diff
@@ -39,6 +41,7 @@
 import java.sql.DriverManager;
 import java.util.Map;
 
+@API(API.Status.EXPERIMENTAL)
 public class SqlInsertTest {
```

Comment:
I don't think we really need API annotations on test classes. It's probably not a big deal if they're there, just unnecessary